__tests__/port.test.js

```diff
@@ -0,0 +1,40 @@
+const Ship = require("../src/Ship.js");
+const Port = require("../src/Port.js");
+
+describe('Port', () => {
+  describe('name, add & remove ship', () => {
+      let port;
```

Comment:
Indentation here should be set to 2 spaces instead of the 4 you use from here to the end of the file